code/modules/power/singularity/particle_accelerator/particle_control.dm

```diff
@@ -208,6 +215,11 @@
 	message_admins("PA Control Computer turned [active ?"ON":"OFF"] by [usr ? key_name_admin(usr) : "outside forces"](<A HREF='?_src_=holder;adminmoreinfo=\ref[usr]'>?</A>) (<A HREF='?_src_=holder;adminplayerobservefollow=\ref[usr]'>FLW</A>) in ([x],[y],[z] - <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[x];Y=[y];Z=[z]'>JMP</a>)",0,1)
 	log_game("PA Control Computer turned [active ?"ON":"OFF"] by [usr ? "[key_name(usr)]" : "outside forces"] in ([x],[y],[z])")
 	if(active)
+		radio.frequency = 1357
+		for (var/obj/machinery/field/containment/C in range(30, src))
+			fieldcount += C		
+		if (fieldcount.len<24)

```

Comment:
I realized that shortly before I opened the PR, I will probably just change the message to reflect that the containment is using a "non-standard" setup and request inspection.